static/src/components/builder/components/DropdownInput/DropdownForm.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <div class="row">
+    <h4>Dropdown Settings</h4>
+    <div class="form-group">
+      <input
+        id="add-label"
+        v-model="labelAdded"
+        type="checkbox"
+      >
+      <label for="add-label">
+        Add Dropdown Label
+      </label>
+      <input
+        v-if="labelAdded === true"
+        v-model="label"
+        class="form-control form-control-sm"
+        type="text"
+      >
+    </div>
+    <label
+      class="block-label"
+    >
+      Answer field name | <span class="label-tip">The field where the worker's answer is stored. Can be JSON path like a.b.c.</span>
+      <input
+        id="pyb-answer"
+        v-model="pybAnswer"
+        class="form-control form-control-sm"
+        type="text"
+      >
+    </label>
+    <label
+      class="block-label"
+    >
+      Initial Value | <span class="label-tip">Must be one of the configured choice values.</span>
+      <select
+        v-model="initialValue"
+        class="form-control form-control-sm"
+      >
+        <option selected />
+        <option
+          v-for="value in values"
+          :key="value"
+        >
+          {{ value }}
+        </option>
+      </select>
+    </label>
+    <hr>
+    <h4>
+      Choices
+    </h4>
+    <div
+        id="choices"
+        class="scroll"
+    >
+      <div
+        v-for="(choice, index) in choiceList"
+        :key="index"
+      >
+        <hr
+          v-if="index != 0"
+          size="100px"
+        >
+        <label>Choice {{ index + 1 }}</label>
+        <button
+          v-if="choiceList.length > 1"
+          :title="`Delete Choice ${index + 1}`"
+          class="btn btn-times-delete pull-right fa fa-times"
+          @click="deleteChoice(index)"
+        /><br>
+        <label
+          class="block-label"
+        >
+          Value | <span class="label-tip">The value to store as the answer when selected.</span>
+          <input
+            id="value"
+            class="form-control form-control-sm"
+            type="text"
+            :value="choice.value"
+            :class="{'danger-validation':getErrors(`choiceList[${index}].value`)}"
```

Comment:
Any form should be validated before any input is touch by the user.  We could use the concept of dirty and then apply validations if the input is dirty and wrong.

Reply:
I think this comment apply for other components where this feature exist.

Reply:
I will work on this in next sprint.